plans/example.fmf

```diff
@@ -0,0 +1,4 @@
+summary: Basic smoke test
+execute:
+    how: tmt
+    script: echo test > testfile
```

Comment:
Can you please add there something different? test is so general. Somethink like https://github.com/sclorg/container-common-scripts/blob/master/test-lib-openshift.sh#L1087

```bash
echo $((11*11)) > testfile
```